src/util/decision_procedure.cpp

```diff
@@ -11,10 +11,4 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "decision_procedure.h"
 
-#include <cassert>
 
-bool decision_proceduret::in_core(const exprt &expr)
-{
-  UNREACHABLE;
-  return true;
-}
```

Comment:
Only half-related to the change: I believe it would be good to explicitly define  the destructor in this .cpp file so that the compiler has an idea where to place the virtual table.